helm/pinot/templates/broker/statefulset.yaml

```diff
@@ -93,6 +93,9 @@ spec:
         livenessProbe:
```

Comment:
since there are multiple probes, shall we change probe configs to `Values.probes.livenessProbe.failureThreshold` / `Values.probes.readinessProbe.failureThreshold` which default to `Values.probes.failureThreshold` but allow customization if needed.

E.g. in StarTree deployment we see that for many environments we need to customize startupProbe and livenessProbe.

Reply:
Sure, I was thinking the same.